### ISSUE TITLE
Tag separators

### ISSUE THREAD

**Issue by thom-cameron** [MaxVerevkin/i3bar-river#28]
Hi. 

Thanks again for making this and sorry for another potentially stupid issue. The "separator"/"separator_width" variables in the config don't seem to be doing anything for me. 

Here's how my i3status-rs stuff looks with a separator:
![image](https://github.com/user-attachments/assets/cac961ed-1285-4b3a-a49d-d12e4af32eeb)

But I don't seem to be able to add similar gaps between my tags:
![image](https://github.com/user-attachments/assets/72b55f29-4140-4d5b-a7d1-768da9693fb5)
(I'd like a blank space between the active tag and the block of inactive ones)

Here's my i3bar-river config. Apologies for all the nix crap
```toml
# colours
background = "#${theme.background.primary}00"
color = "#${theme.colours.light.white}ff"
separator = "#${theme.background.primary}00"
tag_focused_fg = "#${theme.colours.light.white}ff"
tag_focused_bg = "#${theme.accent.primary}ff"
tag_fg = "#${theme.colours.normal.white}ff"
tag_bg = "#${theme.background.secondary}ff"
tag_urgent_fg = "#${theme.colours.light.white}ff"
tag_urgent_bg = "#${theme.accent.alert}ff"
tag_inactive_fg = "#${theme.colours.normal.white}ff"
tag_inactive_bg = "#${theme.background.primary}00"

# font and sizes
font = "${theme.font.mono.name} 10.5"
height = 22
margin_top = 0
margin_bottom = ${toString theme.desktop.padding}
margin_left = ${toString theme.desktop.padding}
margin_right = ${toString theme.desktop.padding}
tags_r = ${toString theme.desktop.corner-radius}
blocks_r = ${toString theme.desktop.corner-radius}
tags_padding = 12
separator_width = 999
blocks_overlap = 0

# misc
position = "bottom"
show_layout_name = true
blend = false

# command
command = "i3status-rs $HOME/.config/i3status-rust/config-bottom.toml"
```
I've got "separator_width" set really high for illustrative purposes. It doesn't seem to make a difference at any size.

Help/advice on this would be much appreciated.

Cheers.

**Comment by MaxVerevkin** [comment]
Correct, tags are not padded right now. `tags_margin` can be added similar to an already existing `tags_padding`.

**Comment by MaxVerevkin** [comment]
Implemented in 3413a20
